AAI/AAIConnectProfile.md

```diff
@@ -314,44 +326,73 @@ upstream.
     3.  A user's withdrawal of this agreement does not need to apply to
         previously generated access tokens.
 
-6.  If an Identity Broker includes Embedded Tokens as part of its /userinfo
-    payload:
+6.  By signing an access token, an Identity Broker asserts that the GA4GH
+    claims that token makes available at the /userinfo endpoint -- not
+    including any Embedded Tokens from other Claim Signatories in those claims
+    -- were legitimately derived from their
+    [Claim Sources](#term-claim-source), and are presented accurately. These
+    assurances apply to any Embedded Tokens that the Identity Broker signs as
+    well when it is acting as an Embedded Claim Signatory.
+            
+    When a Broker provides Embeded Tokens from other Embedded Claim
+    Signatories, it is providing them "as is" (i.e. it provides no additional
+    assurance as to the quality, authenticity, or trustworthiness of the
+    claims from such tokens and any such assurances are made by the issuer of
+    the Embedded Token).
+
+#### Conformance for Embedded Claim Signatories
 
-    1. Embedded Tokens MUST contain one of the following, but not both:
+1.  Embedded Tokens MUST contain one of the following, but not both:
 
-       1.  Header contains `jku`; or
+    1.  Header contains `jku`; or
 
-       2.  Body contains the "openid" scope: has a `scope` claim that contains
-           "openid" as a space-delimited substring.
+    2.  Body contains the "openid" scope: has a `scope` claim that contains
+        "openid" as a space-delimited substring.
 
-    2. If the Embedded Token header contains `jku`:
+2.  If the Embedded Token header contains `jku`:
 
-       1.  The token is not treated as an OIDC access token, but validity
-           checks outlined elsewhere in this specification still apply.
+    1.  The token is not treated as an OIDC access token, but validity
```

Comment:
Just "access token" as OIDC/OAuth terminology can be confusing and nitpicky. because technically isn't it an Oauth access token? then we go down that rabbit hole. Just say "access token".

Reply:
Done